app/views/gardens/show.html.erb

```diff
@@ -3,8 +3,13 @@
 <div class="plants">
   <ul>
     <% @garden.plants.each do |plant| %>
-      <li><%= plant.name %><br />
-      Watering Requirements: <%= plant.times_per_week.round(0) %> times/week</li>
+      <div id="plant-<%= plant.id %>"
```

Comment:
This is will be a conflict with the garden show page tests, but will be easy to resolve